### ISSUE TITLE
Allow appending entities to arrays

### ISSUE THREAD

**Issue by DavidLeoni** [opendatatrentino/OpenDataRise#194]
We can allow some minimal editing of relational arrays in step 4 by allowing appending (middle insertion can be implemented later if we need it)

**Comment by DavidLeoni** [comment]
To fix this one, other new features have been introduced:

### ARRAYS

There are new GREL functions to play with arrays:

```
isArray([1,2,3])

[1,2,3].concat([4, 5])   --> [1,2,3,4,5]

[1,2,3].push(4, 5) --> [1,2,3,4,5]
```

### SEMTEXT

There are new semtext functions to create a semtext with only one meaning (that is, a tag):

```
semtext(String text, String lang, String MeaningStatus)
semtext(Stringtext, String lang, String MeaningStatus, String selectedMeaningUrl,  String selectedMeaningKind)
```

**Comment by DavidLeoni** [comment]
Closed in dev-0.3

Allows appending entities and concepts
